hack/cleanup.sh

```diff
@@ -27,7 +27,7 @@ find ${PREFIX}/usr/bin -type f -exec strip --strip-all {} ';' || true
 # Remove header files, man files, and any other non-runtime dependencies.
```

Comment:
I think we can drop this script almost with `--exclude` directives in dockerfile